imagetest/test_suites/image_validation/cloud_test.go

```diff
@@ -0,0 +1,97 @@
+package imagevalidation
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	chronyService = "chronyd"
+	ntpdService   = "ntp"
+)
+
+var ntpConfig = []string{"/etc/ntp.conf"}
+var chronyConfig = []string{"/etc/chrony.conf", "/etc/chrony/chrony.conf", "/etc/chrony.d/gce.conf"}
+
+// TestNTPService Verify that ntp package exist and configuration is correct.
+// For SUSE, the ntp service is ntpd. For others, the ntp service is chronyd
+func TestNTPService(t *testing.T) {
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("Couldn't get image from metadata")
+	}
+	var servicename string
+	var configPaths []string
+	switch {
+	case strings.Contains(image, "debian-10"):
+	case strings.Contains(image, "debian-11"):
+	case strings.Contains(image, "rhel"):
+	case strings.Contains(image, "centos"):
+	case strings.Contains(image, "sles"):
+		servicename = chronyService
+		configPaths = chronyConfig
+	case strings.Contains(image, "ubuntu"):
+		switch {
+		case strings.Contains(image, "ubuntu-1604"):
+		case strings.Contains(image, "ubuntu-minimal-1604"):
+			servicename = ntpdService
+			configPaths = ntpConfig
+		default:
+			servicename = chronyService
+			configPaths = chronyConfig
+		}
+	default:
```

Comment:
in the comment for this function you suggest ntp is the exception, and is suse only. but here in the code you've said sles uses chrony, and set the default to ntp. if chrony is the more common, we should make that the default, and call out the explicit cases that use ntp

Reply:
Oops, the comment is outdated. Sles-12 use ntpd and chrony. It's complicate across distros. I will update comment.